### DISCUSSION TITLE
All you want

### DISCUSSION THREAD

**Discussion by jery123** [community/community#61625] [Copilot Conversations] [answered]
Hi all,
I'm jery FOTO, actually i am in India for study. Please i want to know more about github copilot

**Comment by Akash1134** [comment, marked as answer]
Hi @jery123 ,

You can learn more about GitHub Copilot from our [quickstart guide](https://docs.github.com/en/copilot/quickstart). 

If you want to explore in detail, check out the series [GitHub Copilot Fundamentals - Understand the AI pair programmer](https://learn.microsoft.com/en-gb/training/paths/copilot/)

Thanks!

**Reply by jery123** [reply]
thanks !
